sdk/js/src/Request.ts

```diff
@@ -40,9 +40,9 @@ export const get = async (url: string): Promise<AxiosResponse> => {
     })
 }
 
-export const getConfigJson = async (environmentKey: string, user: DVCPopulatedUser): Promise<BucketedUserConfig> => {
+export const getConfigJson = async (environmentKey: string, user: DVCPopulatedUser, enableCloudData?: boolean): Promise<BucketedUserConfig> => {
     const queryParams = `${serializeUser(user)}`
-    const url = `${BASE_URL}${HOST}${CONFIG_PATH}?envKey=${environmentKey}${queryParams && '&' + queryParams}`
+    const url = `${BASE_URL}${HOST}${CONFIG_PATH}?envKey=${environmentKey}${enableCloudData ? ('&enableCloudEntityData=' + enableCloudData): ''}${queryParams && '&' + queryParams}`
```

Comment:
to make it easier to add this query param you can see if you can follow the way i did it recoded the query params: https://github.com/DevCycleHQ/js-sdks/pull/148/files#diff-b985bf038e49b8ee77cff70485df18db4d9fd1d6ff6992dd90a1038f9b547393R72

You can then add append it conditionally based on the passed in param `enableCloudData`